subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/PlainTextBuildVerdictWriter.kt

```diff
@@ -0,0 +1,40 @@
+package com.avito.android.build_verdict
+
+import com.avito.utils.logging.CILogger
+import java.io.File
+
+internal class PlainTextBuildVerdictWriter(
+    private val buildVerdictDir: File,
+    private val logger: CILogger
+) : BuildVerdictWriter {
+
+    override fun write(buildVerdict: BuildVerdict) {
+        val dir = buildVerdictDir.apply { mkdirs() }
+        val file = File(dir, buildVerdictFileName)
+        file.createNewFile()
+        file.writeText(
+            """
+Your build FAILED: "${buildVerdict.rootError.message}"
```

Comment:
E.g. how the verdict looks

Your build FAILED: "Execution failed for task ':app:kaptDebugKotlin'."
-------------------------------------------------------

Failed tasks:
1. Task :app:kaptDebugKotlin
Errors:
/private/var/folders/tx/0j4xvb9d36xfgv1h3hd4th580000gn/T/junit16968900557237870342/app/build/tmp/kapt3/stubs/debug/DaggerComponent.java:6: error: [Dagger/MissingBinding] CoffeeMaker cannot be provided without an @Inject constructor or an @Provides-annotated method.
public abstract interface DaggerComponent {
                ^
      CoffeeMaker is requested at
          DaggerComponent.maker()
________________________________________